code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -12,7 +12,9 @@
 	var/projectilesound
 	var/casingtype
 	var/move_to_delay = 2 //delay for the automated movement.
-	var/list/friends = list()
+
+	var/list/friends = list() //List of friends. Accepts both paths (/mob/living/simple_animal/corgi) and mob references.
```

Comment:
Oh man this holds actual mob references? That ain't good it'll break garbage collection for them. Using text refs would be preferrable if possible.